hack/addons/mindthegap-helm-registry/repos.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#=================================================================
+#                 DO NOT EDIT THIS FILE
+#  IT HAS BEEN GENERATED BY /hack/addons/generate-mindthegap-repofile.sh
+#=================================================================
+repositories:
+  cilium:
+    repoURL: https://helm.cilium.io/
+    charts:
+      cilium:
+      - 1.15.0
+  cluster-autoscaler:
+    repoURL: https://kubernetes.github.io/autoscaler
+    charts:
+      cluster-autoscaler:
+      - 9.35.0
```

Comment:
anyway better way to keep it in sync with one in chart https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/637/files#diff-3b9b812f97c38c8ede8337eabf11d60d59b53e6ce54b66db1f2dced13035a5e6R16

Reply:
same here too. https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/637/files#diff-4307f3fa02a9e4bd4c0051d5c9910ce9c8863cb0725f6c4e6bc2f29b7fd3d094R7

can we replace the same from addon.mk file everywhere so that we keep it at once place

Reply:
Thats a good question, the values for repos.yaml and the helm-config.yaml are derived from the makefile.